aiida/parsers/plugins/templatereplacer/parser.py

```diff
@@ -69,13 +69,13 @@ def parse(self, **kwargs):
 
     @staticmethod
     def parse_stdout(filelike):
-        """Parse the sum from the output of the ArithmeticAddcalculation written to standard out.
+        """Parse the content from the output of the TemplatereplacerCalculation written to output file.
```

Comment:
Note that we no longer need the try/catch in this method. The `ValueError` was being caught in case the output could not be cast to an integer. So really this method should just do `return filelike.read()` but that is ridiculous, so might as well get rid of the method entirely and just change line 32 to 
```python
result = handle.read()
```
and be done with it.

Reply:
Good point. Changed.